spinnaker_service/src/main/java/revature/projectFactory/spinnaker/spinnakerServices/PipelineCreation.java

```diff
@@ -40,8 +45,11 @@ public boolean create(PipelineCreationPojo pipline) {
             reader.close();
         }catch(IOException e){
             e.printStackTrace();
+            log.error("Spinnaker Get pipeline creation connection failed", e);
         }
+        log.info("Spinnaker pipeline creation connection is false");
         return false;
+
```

Comment:
remove the logging from this file for now as my other branch was working on it and we added logging as we did thus this will cause a merge conflict.

Reply:
The logger has been deleted. My bad for all the different colors (accidentally clicked on auto-format)